frontend/app/views/fragments/form/submit.scala.html

```diff
@@ -44,3 +47,11 @@
         </div>
     </div>
 </fieldset>
+
+@countryGroup.map{ cg =>
+    @if(Seq("us","um","vi").contains(cg.id.toLowerCase)) {
```

Comment:
@rtyley that might have been fortunate in the case of Guardian Weekly pricing - as it costs more to post to those places and so any pages which resolve prices to readers from those places would have to quote the international (RestOfTheWorld) prices.